plugins/ipstack/icon_ipstack/connection/connection.py

```diff
@@ -10,4 +12,17 @@ def __init__(self):
         self.token = None
 
     def connect(self, params):
-        self.token = params["cred_token"]["secretKey"]
+        self.token = params.get(Input.CRED_TOKEN).get("secretKey")
+
+    def test(self):
+        url = "http://api.ipstack.com/" + "check" + "?access_key=" + self.token + "&output=json"
+        resp = insightconnect_plugin_runtime.helper.open_url(url)
+        dic = json.loads(resp.read())
+        if "error" in dic:
+            code = dic["error"].get("code")
+            if code < 200 or code > 200:
```

Comment:
Wouldn't be better to change that into this?
```suggestion
            if code != 200:
```